curriculum/challenges/english/25-front-end-development/workshop-fruit-search-app/67f8ec758a740e34432222db.md

```diff
@@ -0,0 +1,121 @@
+---
+id: 67f8ec758a740e34432222db
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+Add an `onChange` handler to the search input that updates the `query` state.
+
+# --hints--
+
+You should ensure that the `onChange` handler is defined for the `input` element, and it updates the `query` state properly.
```

Comment:
I can see a lot of campers struggling to get to the right answer.
updating state like this is still going to be new for them.
So the hint and initial description needs to be updated to help them out.

In previous workshops, we have provided examples for them as reminders since they are new to the topic. so that would work in this case. I would suggest studying other workshops that come before this for the wording and examples.